implementation.md

```diff
@@ -2,4 +2,64 @@
 layout: default
 ---
 
-Work in progress. 
+# Sinopia Technology Implementation
+The Sinopia stack uses [Circle-ci][CI] for continuous integration to
+validate the code syntax using [eslint](https://eslint.org/) and test the code base using a combination of
+[Jest](https://jestjs.io/), [Enzyme](https://github.com/airbnb/enzyme), and
+[Puppeteer](https://developers.google.com/web/tools/puppeteer/). [Circle-ci][CI] also
+creates [Docker][DOCK] images on Dockerhub that are automatically deployed to Sinopia's AWS
+development environment.
+
+## Sinopia Linked Data Editor
+A hard fork of the Library of Congress [BIBFRAME Editor](https://github.com/lcnetdev/bfe),
+Sinopia's Linked Data Editor is a [node.js][NODE] project that
+uses the open-source javascript library [React](https://reactjs.org/) to build
+the client-side user interface with supporting [node.js][NODE] libraries like
+[Redux](https://redux.js.org/) for managing the data state of the editor.
+
+The Linked Data Editor also implements a [Swagger REST API][SWAG]
+for search and look-up interactions with third parties like the Questioning
+Authority service from Cornell. The Sinopia Linked Data Editor requires a valid
+[JSON Web Tokens (JWT)][JWT] from users that authenticate through
```

Comment:
Not a Markdown link.

Reply:
Should `Tokens` here be `Token`?

Reply:
It is plural at the https://jwt.io/ website; you're right, in this context should be singular. Fixed in latest commit.